site/content/3.12/index-and-search/indexing/working-with-indexes/multi-dimensional-indexes.md

```diff
@@ -123,11 +155,103 @@ Thus our query would be:
 
 ```aql
 FOR app IN appointments
-    FILTER f <= app.to
-    FILTER app.from <= t
-    RETURN app
+  FILTER f <= app.to
+  FILTER app.from <= t
+  RETURN app
+```
+
+## Prefix fields
+
+Multi-dimensional indexes can accelerate range queries well but they are
+inefficient for queries that check for equality of values. For use cases where
+you have a combination of equality and range conditions in queries, you can use
+the `mdi-prefixed` subtype instead of `mdi`. It has all the features of the
+`mdi` subtype but additionally lets you define one or more document attributes
+you want to use for equality checks. This combination allows to efficiently
+narrow down the search space to a subset of multi-dimensional index data before
+performing the range checking.
```

Comment:
The attribute values in the `prefixFields` can also be non-numeric. Which usually was a huge problem with the old `mdi`.